### PR TITLE
Fix/overflow view page

### PR DESCRIPTION
**Pull request by imrishabh18**
Closes - #151 

https://github.com/user-attachments/assets/88644902-28a3-4406-a206-d6a79e219dfa

### CODE REVIEW

File: src/components/PreviewContent.tsx [hunk 1]
```diff
@@ -80,7 +80,7 @@ export const PreviewContent = ({
         <Tabs
           value={activeTab}
           onValueChange={setActiveTab}
-          className="flex-grow flex flex-col"
+          className="flex-grow flex flex-col sticky top-0 h-screen"
```
Comment (seveibar) [inline src/components/PreviewContent.tsx:83]: hmm is this needed? the container already has sticky

**Comment by seveibar** [comment]
@imrishabh18 future pushes will deploy the preview

**Comment by seveibar** [comment]
I want to play with this PR, for some reason the video doesn't load and I'm confused by the sticky and overflow parameters

File: src/components/PreviewContent.tsx (hunk 1 above)
Comment (imrishabh18) [reply src/components/PreviewContent.tsx:83]: Yes, that container sticky even when provided with the `h-screen` it doesn't make it stick. I think we can remove that div as a whole

https://github.com/user-attachments/assets/75a88b52-774e-46fd-92e9-41881bfdad8f

**Comment by seveibar** [comment]
@imrishabh18 instead of having this overflow with scroll, can we make it expand to the full size of the page the same way it does on the editor page?


image

File: src/components/PreviewContent.tsx (hunk 1 above)
Comment (seveibar) [reply src/components/PreviewContent.tsx:83]: it might be my computer but the videos you're posting are 0s, is it just me?

Comment (seveibar) [reply src/components/PreviewContent.tsx:83]: I still don't understand what this line fixes though, it's already in a sticky container. Maybe the video would help (let me know if it's on my end)

**Comment by imrishabh18** [comment]
@seveibar You mean something like this?


https://github.com/user-attachments/assets/1f9e7376-28d1-43a0-9fcc-76df1c9324bd

**Comment by seveibar** [comment]
> @seveibar You mean something like this?
> 
>  Screen.Recording.2024-11-03.at.3.16.52.AM.mov

Yep! I think that is more natural- it's what github does after all!

File: src/components/PreviewContent.tsx (hunk 1 above)
Comment (imrishabh18) [reply src/components/PreviewContent.tsx:83]: I think on my end, the video recorded was playing in my machine. But on the upload preview it doesn't

Comment (seveibar) [reply src/components/PreviewContent.tsx:83]: Very strange. I use [Kap](https://getkap.co/) for recording fwiw

**Pull request merged**